vignettes/GOstatsHyperG.Rmd

```diff
@@ -2,7 +2,6 @@
 title: "How To Use GOstats Testing Gene Lists for GO Term Association"
 author:
   - name: S. Falcon and R. Gentleman
```

Comment:
Sorry I didn't catch this before, but separate the 2 authors on this line into separate lines